cli/bin/setup

```diff
@@ -1,3 +1,14 @@
 #!/usr/bin/env bash
-echo "disabled until ZMQ is removed"
-# npm install --loglevel error --depth 0
+
+npm install --loglevel error --depth 0
+(cd ../exo-add && bin/setup && node_modules/.bin/build && echo 'exo-add √')
```

Comment:
The only problem with using Morula here is that this is inside of `cli`s `bin/setup` and if we tell Morula to execute `bin/setup` for each sub-repo, we will get caught in a loop. I could change the name of this script to something else to resolve this?